### PR TITLE
[DEV-1692] Allow log level to be configured using environment variable

### PR DESCRIPTION
**Pull request by YS-L**
## Description

This allows logger's log level to be configured using the environment variable `LOG_LEVEL`. Previously, the log level was set based on the configurations yaml file, but that file is no longer mounted into the containers as it should contain only client side configurations.

## Type of Change

Label this pull request to place it under the correct category in Release Notes:

|        **Pull Request Label**         | **Category in Release Notes** |
|:-------------------------------------:|:-----------------------------:|
|       `enhancement`, `feature`        |          🚀 Features          |
| `bug`, `refactoring`, `bugfix`, `fix` |    🔧 Fixes & Refactoring     |
|       `build`, `ci`, `testing`        |    📦 Build System & CI/CD    |
|              `breaking`               |      💥 Breaking Changes      |
|            `documentation`            |       📝 Documentation        |
|            `dependencies`             |    ⬆️ Dependencies updates    |

## Checklist

- [ ] I have read the [`CODE_OF_CONDUCT.md`](https://github.com/featurebyte/featurebyte/blob/main/CODE_OF_CONDUCT.md) and [`CONTRIBUTING.md`](https://github.com/featurebyte/featurebyte/blob/main/CONTRIBUTING.md) guides.
- [ ] I have written tests for the changes made.
- [ ] I have written docstrings in [NumpyDoc format](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)
- [ ] I have labeled my Pull Request correctly

### CODE REVIEW

File: docker/entrypoint.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ _main() {
       celery --app featurebyte.worker.start.celery worker -Q io_task,io_task:1,io_task:2,io_task:3 --loglevel=INFO --pool=gevent -c 1000
     elif [ "$1" = 'server' ]; then
       python /scripts/migration.py
-      uvicorn featurebyte.app:app --host=$API_HOST --port=$API_PORT --timeout-keep-alive=300 --log-level=$LOG_LEVEL
+      uvicorn featurebyte.app:app --host=$API_HOST --port=$API_PORT --timeout-keep-alive=300 --log-level=info
```
Comment (kchua78) [inline docker/entrypoint.sh:52]: Do we want to fix the log level for uvicorn?

Comment (YS-L) [reply docker/entrypoint.sh:52]: Is it useful to be able to override the log level for uvicorn? If so, I think we can add a different variable to control that. Same for workers' log level which is currently also fixed as INFO.

Comment (kchua78) [reply docker/entrypoint.sh:52]: I guess not. Let's leave it as info then.

**Comment by YS-L** [comment]
Timing seems to be back to normal now. Will merge.

**Pull request merged**